libbeat/docs/processors-using.asciidoc

```diff
@@ -1,25 +1,24 @@
 [[defining-processors]]
 === Define processors
 
-You define processors in the +{beatname_lc}.yml+ file to filter and enhance the
-data before sending events to the configured output.
-
-To define a processor, you specify the processor name, an optional condition,
-and a set of parameters:
+You can use processors to filter and enhance data before sending it to the
```

Comment:
Now that we have some configs in the modules.d directory, telling users that they specify processors in the {beatname_lc}.yml file seems misleading. I think it's pretty clear that these are configuration settings by the context, no?